src/modules/product/Product/Product.tsx

```diff
@@ -175,7 +179,8 @@ class Product extends React.Component<
             style={styles.productMainScreen}
           >
             <Text style={styles.infoTitle}>
-              {product.name} {brand.name} {"\n"} {activeSubProduct.article}
+              {product.name} {product.brand.name} {"\n"}{" "}
```

Comment:
Replace with template literal `${product.name} ${product.brand.name}\n  ${activeSubProduct.article}` 
or better get rid from `\n` and replace with 

```
<View>
<Text>..</Text>
<Text>..</Text>
</View>
```